docs/getting-started.rst

```diff
@@ -128,6 +128,8 @@ Required columns in input data
 +---------------------+--------------------------------------------------------+-------+
 | heizkost_m          | Monthly heating expenses                               | float |
 +---------------------+--------------------------------------------------------+-------+
+| mietstufe           | Level of rent in community                             | int   |
```

Comment:
maybe add "3 refers to average rent"

Reply:
Well that depends highly on the number of possible mietstufen in that year?

Reply:
Since it's now mandatory we have to give some guidelines, i.e. the possible range. and "if in doubt, choose 3". Before 1985, Mietstufe 3 is for cities > 500.000 inhabitants.

Reply:
Added some guideline. But I think will need sooner or later a more detailed documentation anyway. Then we can include such guidelines. We probably will have more, as not all datasets contain all variables.